src/Nri/Ui/RadioButtonDotless/V1.elm

```diff
@@ -0,0 +1,355 @@
+id id_ =
+    Attribute <| \config -> { config | id = Just id_ }
+
+
+{-| Use this helper to add custom attributes.
+
+Do NOT use this helper to add css styles, as they may not be applied the way
+you want/expect if underlying styles change.
+Instead, please use the `containerCss` or `labelCss` helper.
+
+-}
+custom : List (Html.Attribute Never) -> Attribute value msg
+custom attributes =
+    Attribute <| \config -> { config | customAttributes = config.customAttributes ++ attributes }
+
+
+{-| Set the "data-nri-description" attribute
+-}
+nriDescription : String -> Attribute value msg
+nriDescription description =
+    custom [ AttributeExtra.nriDescription description ]
+
+
+{-| See Cypress best practices: <https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements>
+-}
+testId : String -> Attribute value msg
+testId id_ =
+    custom [ AttributeExtra.testId id_ ]
+
+
+applyConfig : List (Attribute value msg) -> Config value msg -> Config value msg
+applyConfig attributes beginningConfig =
+    List.foldl (\(Attribute update) config -> update config)
+        beginningConfig
+        attributes
+
+
+emptyConfig : Config value msg
+emptyConfig =
+    { id = Nothing
+    , onSelect = Nothing
+    , width = UnboundedWidth
+    , textAlign = TextAlignCenter
+    , size = Medium
+    , containerCss = []
+    , labelCss = []
+    , customAttributes = []
+    }
+
+
+{-| View the single dotless radio button
+-}
+view :
+    { label : String
+    , name : String
+    , value : value
+    , valueToString : value -> String
+    , selectedValue : Maybe value
+    }
+    -> List (Attribute value msg)
+    -> Html msg
+view { label, name, value, valueToString, selectedValue } attributes =
+    let
+        config =
+            applyConfig attributes emptyConfig
+
+        idValue =
+            case config.id of
+                Just specificId ->
+                    specificId
+
+                Nothing ->
+                    AttributeExtra.safeId (name ++ "-" ++ valueToString value)
+
+        isChecked =
+            selectedValue == Just value
+    in
+    span
+        [ Attributes.class "Nri-RadioButton-Dotless"
+        , css
+            ([ position relative
+             , display inlineBlock
+             , textAlign center
+             , pseudoClass "focus-within" FocusRing.styles
+             , case config.width of
+                UnboundedWidth ->
+                    width unset
+
+                FillContainerWidth ->
+                    width (pct 100)
+             ]
+                ++ config.containerCss
+            )
+        ]
+        [ radio name
+            (valueToString value)
+            isChecked
+            ([ Attributes.id idValue
+             , Attributes.class "Nri-RadioButton-HiddenRadioInput"
+             , case config.onSelect of
+                Just onSelect_ ->
+                    Events.onClick (onSelect_ value)
+
+                Nothing ->
+                    AttributeExtra.none
```

Comment:
I know this isn't something that `RadioButton` does particularly well, but I wonder if we could disable the `RadioButtonDotless` when no `onSelect` is present?

If not, maybe in the `preview`, we could pass through custom attributes of `tabindex=0` and `aria-disabled=true`... 

Right now, these new radio buttons are tabstops on the "catalog" view, is all.